FAIL ./yatze.test.ts
  Chance
    ✓ roll 5 dice (2 ms)
  Yatze
    ✓ roll 5 dice successfully
    ✓ roll 5 dice unsuccessfully (1 ms)
  Ones
    ✓ roll 5 dice successfully
    ✓ roll 5 dice unsuccessfully
  Twos
    ✓ roll 5 dice successfully (1 ms)
    ✓ roll 5 dice unsuccessfully
  Threes, Fours, Fives, Sixes
    ✓ roll 5 dice successfully (1 ms)
    ✓ roll 5 dice unsuccessfully
  Pairs
    ✓ roll 5 dice successfully
    ✕ roll 5 dice unsuccessfully (2 ms)

  ● Pairs › roll 5 dice unsuccessfully

    expect(received).toEqual(expected) // deep equality

    Expected: 0
    Received: 8

      59 |   })
      60 |       it('roll 5 dice unsuccessfully', () => {
    > 61 |     expect(roll([3,3,3,4,1],"Pairs")).toEqual(0);
         |                                       ^
      62 |   })
      63 | });

      at Object.<anonymous> (yatze.test.ts:61:39)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 10 passed, 11 total
Snapshots:   0 total
Time:        2.864 s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ test: `jest --maxWorkers=2`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sandbox/.npm/_logs/2025-05-04T21_43_18_609Z-debug.log
